ScreenCapture.NET/Model/Display.cs

```diff
@@ -1,93 +1,92 @@
-        /// <param name="width">The with of the <see cref="Display"/>.</param>
-        /// <param name="height">The height of the <see cref="Display"/>.</param>
-        /// <param name="graphicsCard">The <see cref="GraphicsCard"/> this <see cref="Display"/> is connected to.</param>
-        public Display(int index, string deviceName, int width, int height, GraphicsCard graphicsCard)
-        {
-            this.Index = index;
-            this.DeviceName = deviceName;
-            this.Width = width;
-            this.Height = height;
-            this.GraphicsCard = graphicsCard;
-        }
-
-        #endregion
-
-        #region Methods
-        
-        /// <summary>
-        /// Determines whether this <see cref="Display"/> equals the given one.
-        /// </summary>
-        /// <param name="other">The <see cref="Display"/> to compare.</param>
-        /// <returns><c>true</c> if the specified object is equal to the current object; otherwise, <c>false</c>.</returns>
-        public bool Equals(Display other) => Index == other.Index;
-
-        /// <inheritdoc />
-        public override bool Equals(object? obj) => obj is Display other && Equals(other);
-
-        /// <inheritdoc />
-        public override int GetHashCode() => Index;
-
-        /// <summary>
-        /// Determines whether two <see cref="Display"/> are equal.
-        /// </summary>
-        /// <param name="left">The first value.</param>
-        /// <param name="right">The second value.</param>
-        /// <returns><c>true</c> if the two specified displays are equal; otherwise, <c>false</c>.</returns>
-        public static bool operator ==(Display left, Display right) => left.Equals(right);
-
-        /// <summary>
-        /// Determines whether two <see cref="Display"/> are not equal.
-        /// </summary>
-        /// <param name="left">The first value.</param>
-        /// <param name="right">The second value.</param>
-        /// <returns><c>true</c> if the two specified displays are not equal; otherwise, <c>false</c>.</returns>
-        public static bool operator !=(Display left, Display right) => !(left == right);
-
-        #endregion
+        this.Index = index;
+        this.DeviceName = deviceName;
+        this.Width = width;
+        this.Height = height;
+        this.GraphicsCard = graphicsCard;
     }
-}
+
+    #endregion
+
+    #region Methods
+        
+    /// <summary>
+    /// Determines whether this <see cref="Display"/> equals the given one.
+    /// </summary>
+    /// <param name="other">The <see cref="Display"/> to compare.</param>
+    /// <returns><c>true</c> if the specified object is equal to the current object; otherwise, <c>false</c>.</returns>
+    public bool Equals(Display other) => Index == other.Index;
+
+    /// <inheritdoc />
+    public override bool Equals(object? obj) => obj is Display other && Equals(other);
+
+    /// <inheritdoc />
+    public override int GetHashCode() => Index;
```

Comment:
I was having some issues with equality comparisons on the Display objects, until I checked on the object itself, even though the issue was obvious, it took a while to make sense. The first display will always have a value of zero on the index, and so will an empty display object, which makes the first display object, and an empty display object, to be always equal. I also am not sure how directx/opengl/vulkan or any other rendering library handles the indexing of Displays, if it's related to all the displays, or if they are indexed on a graphics card connection basis, but I think the following changes made sense.

```suggestion
    public bool Equals(Display other) => Index == other.Index && GraphicsCard == other.GraphicsCard;

    /// <inheritdoc />
    public override bool Equals(object? obj) => obj is Display other && Equals(other);

    /// <inheritdoc />
    public override int GetHashCode() => Index + GraphicsCard.GetHashCode();
```

For the same previous reasons, I thought about using the DeviceName instead of the GraphicsCard, since I'm not sure if the first display on separate graphics cards will as well be always named \\.\Display1, or not, but I'm sure that the graphics card will be different.

I also thought while writing this, that the default value for the index may be changed to -1, but I don't think that it would be as robust nor fool and future proof as my changes.